To workaround the following error

  In file included from src/ui/mainwindow.cpp:2:
  build/ui/ui_mainwindow.h:1077:25: error: no member named 'setPlaceholderText' in
        'QPlainTextEdit'
          outputTextEdit->setPlaceholderText(QApplication::translate("MainWind...
          ~~~~~~~~~~~~~~  ^
  1 error generated.
  *** Error code 1

--- src/ui/mainwindow.ui~
+++ src/ui/mainwindow.ui
@@ -222,9 +222,6 @@ QScrollBar::add-page:horizontal, QScroll
         <property name="readOnly">
          <bool>true</bool>
         </property>
-        <property name="placeholderText">
-         <string>Baka MPlayer loaded...</string>
-        </property>
        </widget>
       </widget>
       <widget class="QWidget" name="playlistLayoutWidget" native="true">
